### DISCUSSION TITLE
Quick navigate between items in a collection

### DISCUSSION THREAD

**Discussion by cs1m0n** [directus/directus#16553] [Feature Requests]
Would it be possible to have arrow navigation between the items in the Admin dashboard?
Instead of go in - go out - scroll down - select the next/previous item - go into the next item; simply click on the next/previous arrow.

**Comment by azrikahar** [comment]
Interesting idea 👍 I believe one alternative suggestion that gained a bit more traction would be #3843 where clicking the items will open them in a drawer rather than navigating to them.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
